Yoast/wordpress-seo
# Removes penalizing for H1 that is not at the beginning of the text body

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Removes penalising for H1 that is not at the beginning of the text body.
* [shopify-seo] Removes penalising for H1 that is not at the beginning of the text body.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

**Test in Block editor and in Elementor**
* Create a post and add some text and a title
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Add an H1 block at the beginning of the text body
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Move the H1 block so that it's not at the beginning of the text body
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Add another H1 block in the text body
* Confirm that `Single title` assessment is triggered, and you can see this feedback: `H1s should only be used as your main title. Find all H1s in your text that aren't your main title and change them to a lower heading level!`
* Click the eye icon and confirm that all H1s are highlighted


**Test in Shopify**
Note: link `wordpress-seo` to `shopify-seo/app` using `yarn link` before building the app (see [this documentation](https://github.com/Yoast/shopify-seo/blob/main/doc/run-the-app.md#embedded-development-of-admin-ui--wordpress-seo-packages))

* Create a product and add some text and a title
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Add an H1 at the beginning of the text body
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Move the H1 so that it's not at the beginning of the text body
* Confirm that `Single title` assessment is not triggered under SEO analysis
* Add another H1 in the text body
* Confirm that `Single title` assessment is triggered, and you can see this feedback: `H1s should only be used as your main title. Find all H1s in your text that aren't your main title and change them to a lower heading level!`
* Click the eye icon and confirm that all H1s are highlighted


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-1380
